ui-tests/package.json

```diff
@@ -5,8 +5,9 @@
   "private": true,
   "scripts": {
     "start": "nbmerge-web -p 41000 data/default/center.ipynb data/default/left.ipynb data/default/right.ipynb ",
-    "test": "npm playwright test",
-    "test:update": "npm playwright test --update-snapshots"
+    "test": "npx playwright test",
+    "test:update": "npx playwright test --update-snapshots",
+    "playwright": "npx playwright"
```

Comment:
This is what makes the `npm run playwright` script work. Alternatively we could provide a custom `update_script` pointing to `test:update` on `jupyterlab/maintainer-tools/.github/actions/update-snapshots@v1` step.